salt/backups/backup_tracking.sls

```diff
@@ -0,0 +1,18 @@
+{% set edx_tracking_bucket = 'odl-residential-tracking-data' %}
+{% set edx_tracking_bucket_creds = salt.vault.read('aws-mitx/creds/read-write-{bucket}'.format(bucket=edx_tracking_bucket)) %}
+{% set edx_tracking_local_folder = '/edx/var/log/tracking' %}
+{% set instance_id = salt.environ.get('MINION_ID') %}
+
+tar_tracking_data:
+  cmd.run:
+    - name: 'tar -czf edx_tracking_{{ instance_id }}.tgz *'
+    - cwd: {{ edx_tracking_local_folder }}
```

Comment:
You can use the `module.run` state to execute the `archive.tar` execution module as an alternative to using `cmd.run`. If you do stick with `cmd.run` then you should add the `creates` parameter so that if it gets run multiple times it will short circuit on recreating the archive.

Reply:
Spent a good amount of time yesterday afternoon looking at docs and trying to get `module.run` to execute `archive.tar` but couldn't get it to work for me and that's when I switched to using `cmd.run`